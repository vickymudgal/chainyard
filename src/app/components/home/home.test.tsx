import * as React from 'react';
import {mount} from 'enzyme';

import HomeComponent from './home.component';

describe('HomeComponent', (): void => {
  it('should display HELLO_WORD', (): void => {
    const component = mount(<HomeComponent/>);
    // verify it has HELLO WORLD
    // HELLO_WORLD is not translated as we have mocked the i18n library
    expect(component.find('h3').text()).toEqual('HELLO_WORD');
    expect(mount).toMatchSnapshot();
  });
});
