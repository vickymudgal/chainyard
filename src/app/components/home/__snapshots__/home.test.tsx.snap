// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should display HELLO_WORD 1`] = `[Function]`;
